### DISCUSSION TITLE
Option to provide Spotify Code

### DISCUSSION THREAD

**Discussion by monsterror** [jac0b-w/album-art-wallpaper#33] [Ideas]
It'd be great if there was an option within Settings to provide both the album art _and_ the Spotify Code, so others can scan and get the song's info.

I use spotify for events, and make custom playlists each week for them. Now that I've started using this app, I'd like to extend my display to show the album art, and it would be great if the audience could also scan to save the song if they'd like.

https://www.spotifycodes.com/#create

**Comment by jac0-b** [comment]
For furture reference: https://github.com/spotify/web-api/issues/519#issuecomment-618114678
